index.html

```diff
@@ -17,14 +17,18 @@
 <body>
     <header class="navbar">
         <a class="logo" href="#">My Logo</a>
-        <nav>
+        <nav class="deskNav">
+            <ul class="navbar-items">
+                <li><a href="#">Portofolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
             <img class="hamburg" src="assets/hamburg.svg" alt="hamburg" />
         </nav>
     </header>
 
     <section class="header">
         <div class="intro">
-
             <h1 class="title">I’m Atif<br>
                 Glad to see you!</h1>
             <p class="description">I’m a software developer! I can help you build a product , feature or website Look
```

Comment:
- [optional] nice job making the changes. please kindly add a background image of the hero section of the mobile version as well. this will help make your work more professional and align with the figma version.
your design
![Screenshot (819)](https://user-images.githubusercontent.com/65298048/220901595-c1182656-0e35-4162-96d6-5e26fa6ac401.png)
figma
![Screenshot (820)](https://user-images.githubusercontent.com/65298048/220901817-22b94c34-507b-4d4f-9184-03e92a780e03.png)